libraries/protocol/include/steem/protocol/steem_required_actions.hpp

```diff
@@ -16,8 +18,76 @@ namespace steem { namespace protocol {
    };
 #endif
 
+#ifdef STEEM_ENABLE_SMT
+   struct smt_refund_action : public base_operation
+   {
+      asset_symbol_type symbol;
```

Comment:
This is minor, but let's put contributor first. Our json output will be in field order and I think it makes more sense to most people to their the account name first.